_targets.R

```diff
@@ -198,6 +198,7 @@ list(
           add_na_flag() %>%
           add_repeat_flag() %>%
           add_suspect_flag() %>%
+          # we should also be incorporating the add_malfuntion_flag() here no?
```

Comment:
Still not totally sure.... Ideally, we wouldn't need this function at all, since the malfunction data should be flagged with our other functions..

Reply:
hmm. I think that even in an ideal scenario where these are caught I don't think it would hurt to flag them again?